saleor/graphql/meta/permissions.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any, List
+
+from graphql_jwt.exceptions import PermissionDenied
+
+from ...account import models as account_models
+from ...core.permissions import (
+    AccountPermissions,
+    BasePermissionEnum,
+    CheckoutPermissions,
+    OrderPermissions,
+    ProductPermissions,
+)
+
+
+def no_permissions(_info, _object_pk: Any) -> List[BasePermissionEnum]:
+    return []
+
+
+def public_user_permissions(info, user_pk: int) -> List[BasePermissionEnum]:
```

Comment:
Some brief docstrings or comments would help here to understand the idea behind "public user permissions", it's purpose etc.